README.rst

```diff
@@ -882,10 +882,9 @@ Build it:
 .. code:: bash
 
    git submodule update --recursive
-   mkdir .build
-   virtualenv --python=python3 .venv
-   .venv/bin/pip install -e .
-   .venv/bin/pip install -r dev-requirements.txt
+   virtualenv --python=python3 .build/venv
```

Comment:
You don't like the .venv folder?

Reply:
No, you didn't. All the tests are using `.build/venv/bin/generate_tiles`since f7baebaaec660c8c1942170ba8238810e6e809dc